x-pack/test/functional/apps/ml/anomaly_detection/lens_to_ml.ts

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObject, getPageObjects }: FtrProviderContext) {
+  const ml = getService('ml');
+  const dashboardPanelActions = getService('dashboardPanelActions');
+  const testSubjects = getService('testSubjects');
```

Comment:
We try to avoid using test subjects directly in our test files but want to wrap that usage in service methods. That makes it easier to reuse and maintain. Some of them might fit into the already existing `ml.dashboardEmbeddables` service, or we could create a new service where we group the lens specific actions and assertions.

Reply:
https://github.com/elastic/kibana/pull/133899/commits/b05cf28b33b439400f16dcc749d72e7edac877f0